tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -21,23 +21,21 @@ import (
 	"istio.io/istio/tests/common/jwt"
 )
 
-func TestMTlsWithAuthNPolicy(t *testing.T) {
+// TODO(diemtvu): remove this test suite. This has been already modernize and well covered
+// under ./tests/integration/security/
+func TestMTlsWithAlphaAuthNPolicy(t *testing.T) {
 	if tc.Kube.AuthEnabled {
 		// mTLS is now enabled via CRDs, so this test is no longer needed (and could cause trouble due
 		// to conflicts of policies)
 		// The whole authn test suites should be rewritten after PR #TBD for better consistency.
 		t.Skipf("Skipping %s: authn=true", t.Name())
 	}
-	// Define the default permissive global mesh resource.
-	globalPermissive := resource{
-		Kind: "MeshPolicy",
-		Name: "default",
-	}
-	// This policy will remove the permissive policy and enable mTLS mesh policy.
+	// Add global alpha mesh policy to enable mTLS. There should not be no beta nor alpha added
+	// in the test setup, so nothing need to be removed.
 	globalCfg := &deployableConfig{
 		Namespace:  "", // Use blank for cluster CRD.
 		YamlFiles:  []string{"testdata/authn/v1alpha1/global-mtls.yaml.tmpl"},
-		Removes:    []resource{globalPermissive},
```

Comment:
These tests are only using the legacy helm stuff, which didn't change now? Do we still need them? Same applies to the dashboard

I am close to removing all of this by the way

Reply:
Yes, we should remove them (I leave a TODO for that). Just don't want to make that change in this PR to make sure it doesn't break any other regression.

Reply:
But nothing in this PR impacts this at all. the tests under tests/e2e use install/, not manifests/, so I don't see why we need to change anything? Maybe I am missing something though

Reply:
Hmm. May be you are right. I saw the error in the first presubmit run when I made the changes in both. I can try to revert to see what happens